api/src/main/java/cz/fi/muni/pa165/dto/MushroomDTO.java

```diff
@@ -1,10 +1,177 @@
 package cz.fi.muni.pa165.dto;
 
+import cz.fi.muni.pa165.enums.MushroomType;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Created by Michal Kysilko 436339 , on 23.11.16.
+ *
+ * @author Michal Kysilko 436339
+ */
+
 public class MushroomDTO implements Comparable<MushroomDTO> {
 
+    public static final int BASE_YEAR = 1970;
+
+    private Long id;
+
+    private String name;
+
+    private MushroomType type;
+
+    private Date fromDate;
+
+    private Date toDate;
+
+    private String description;
+
+    public MushroomDTO() {}
+
+    public long getId() {
+        return this.id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public MushroomType getType() {
+        return this.type;
+    }
+
+    public void setType(MushroomType type) {
+        this.type = type;
+    }
+
+    public Date getFromDate() {
+        return new Date(this.fromDate.getTime());
+    }
+
+    /**
+     * Sets this date which is exclusive to the interval, doesn't use the Time of the object,
+     * <p>
+     * This method also recalculates toDate of this object
+     *
+     * @param fromDate date object (Month and Day are used only)
+     */
+    public void setFromDate(Date fromDate) {
```

Comment:
This is not needed. Dozer uses setters for mapping properties. And API should know what data it is getting/mapping from/to DTO